docs/src/basics/FAQ.md

```diff
@@ -58,6 +59,13 @@ because that is what lower-level packages implement.
 Fixed quadrature rules from other packages can be used with `QuadratureRule`.
 Otherwise, feel free to open an issue or pull request.
 
+## My integrand works with algorithm X but fails on algorithm Y
+
+While bugs are not out of the question, certain algorithms, especially those implemented in C, are not compatible with arbitrary Julia types and have to return specific numeric types or arrays thereof.
+In some cases, such as [`ArblibJL`](@ref), it is also expected that the integrand work with a custom quadrature point type.
+Moreover, some algorithms, such as [`VEGAS`](@ref), only support scalar integrands.
+For more details see the [solver page](solvers).
```

Comment:
I presume you mean

```
┌ Error: invalid local link/image: file does not exist in src/basics/FAQ.md
│   link =
│    @ast MarkdownAST.Link("solvers", "") do
│      MarkdownAST.Text("solver page")
│    end
│    
└ @ Documenter ~/.julia/packages/Documenter/1HwWe/src/utilities/utilities.jl:44
```

In which case it's this one.

Reply:
It should likely just be
```suggestion
For more details see the [solver page](@id solvers).
```

Reply:
Should be `@ref`:

```suggestion
For more details see the [solver page](@ref solvers).
```

Reply:
Yeah... 12 hour time difference with NZ will do that.